eng/service.proj

```diff
@@ -44,6 +44,14 @@
       <MgmtExcludePaths Include="$(MSBuildThisFileDirectory)..\sdk\hdinsight\Microsoft.Azure.HDInsight.Job\**\*.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I'm not a fan of this approach. If we feel it is necessary to unblock folks quickly than please go ahead but I feel like we can do this better using the service.proj approach and ProjectsToIncludeBySDKType